test/index.js

```diff
@@ -155,7 +156,7 @@ function transform(filePath, options = {}, {multiplePasses = false} = {}) {
         return [plugin, {
             ...BASE_OPTIONS,
             ...options,
-        }];
+        }, uuidv1()];
```

Comment:
somewhat dirty fix, need to add random string suffix to allow plugin to run more than once